src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -674,12 +674,25 @@
       CacheFilePath="$(_AOTCompilerCacheFile)"
       LLVMDebug="dwarfdebug"
       LLVMPath="$(EmscriptenUpstreamBinPath)"
+      CollectCompiledMethods="$(WASMStripIL)"
```

Comment:
Documentation for the property should be added to WasmApp.targets. A default value should also be added.

Also, there is already an unused property named WasmStripAOTAssemblies which seems to do the same.

Reply:
Yeah, this can go in https://github.com/dotnet/runtime/blob/bb38848e9877dc0b43860dc219ebe36fc04a6bb3/src/mono/wasm/build/WasmApp.Native.targets#L708-L729

Reply:
Setting this flag to true will enable the feature trimming away AOT compiled methods, instead of the whole assembly. `WasmStripAOTAssemblies` does match the meaning of this feature. I could rename it to `WasmStripAOTMethods`, if that sounds better?

Reply:
Fixed.